service/application/src/main/java/com/solace/maas/ep/event/management/agent/processor/CommandLogStreamingProcessor.java

```diff
@@ -32,6 +33,7 @@
 
 @Component
 @Slf4j
+@ConditionalOnProperty(name = "event-portal.gateway.messaging.standalone", havingValue = "false")
 public class CommandLogStreamingProcessor {
```

Comment:
Only used for config push and the other config push related beans have this conditional. Without it, it tries to find the dependent beans but finds none when not in standalone mode.